x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/filters/filters_section.tsx

```diff
@@ -0,0 +1,125 @@
+ * 2.0.
+ */
+
+import React, { type Dispatch, memo, type SetStateAction, useCallback, useMemo } from 'react';
+import type { Filter, TimeRange } from '@kbn/es-query';
+import { TableId } from '@kbn/securitysolution-data-table';
+import type { FilterGroupHandler } from '@kbn/alerts-ui-shared';
+import type { DataView, DataViewSpec } from '@kbn/data-views-plugin/common';
+import { PageFilters } from './page_filters';
+import type { AssigneesIdsSelection } from '../../../../common/components/assignees/types';
+import { useDataTableFilters } from '../../../../common/hooks/use_data_table_filters';
+import { useDeepEqualSelector } from '../../../../common/hooks/use_selector';
+import { useGlobalTime } from '../../../../common/containers/use_global_time';
+import { inputsSelectors } from '../../../../common/store/inputs';
+import {
+  buildAlertAssigneesFilter,
+  buildShowBuildingBlockFilter,
+  buildThreatMatchFilter,
+} from '../../alerts_table/default_config';
+import type { Status } from '../../../../../common/api/detection_engine';
+
+export interface FiltersSectionProps {
+  /**
+   * List of assignees retrieved from the assignees button on the alert page
+   */
+  assignees: AssigneesIdsSelection[];
+  /**
+   * Data view used for the alerts page
+   */
+  dataView: DataView | DataViewSpec;
+  /**
+   * Callback to set the page filter handler for the alerts page to allow a refresh after the table has reloaded
+   */
+  setPageFilterHandler: Dispatch<SetStateAction<FilterGroupHandler | undefined>>;
+  /**
+   * Callback to set the page filters for the alerts page as it's also used in the GroupedTable component
+   */
+  setPageFilters: Dispatch<SetStateAction<Filter[] | undefined>>;
+  /**
+   * Callback to set the status filter to the alerts page as it's also used in the GroupedTable component
+   */
+  setStatusFilter: Dispatch<SetStateAction<Status[]>>;
+}
+
+/**
+ * UI section of the alerts page that renders the page filters
+ */
+export const FiltersSection = memo(
+  ({
+    assignees,
+    dataView,
+    setPageFilterHandler,
+    setPageFilters,
+    setStatusFilter,
+  }: FiltersSectionProps) => {
+    const getGlobalFiltersQuerySelector = useMemo(
+      () => inputsSelectors.globalFiltersQuerySelector(),
+      []
+    );
+    const getGlobalQuerySelector = useMemo(() => inputsSelectors.globalQuerySelector(), []);
+    const query = useDeepEqualSelector(getGlobalQuerySelector);
+    const filters = useDeepEqualSelector(getGlobalFiltersQuerySelector);
+
+    const { to, from } = useGlobalTime();
+
+    const { showBuildingBlockAlerts, showOnlyThreatIndicatorAlerts } = useDataTableFilters(
+      TableId.alertsOnAlertsPage
+    );
+
+    const topLevelFilters = useMemo(() => {
+      return [
+        ...filters,
+        ...buildShowBuildingBlockFilter(showBuildingBlockAlerts),
+        ...buildThreatMatchFilter(showOnlyThreatIndicatorAlerts),
+        ...buildAlertAssigneesFilter(assignees),
```

Comment:
I already suggested to remove `buildAlertAssigneesFilter` in another comment. I'm wondering also if we need `buildShowBuildingBlockFilter` and `buildThreatMatchFilter` here? At the time of this PR, we do not have an attacks table, and we don't have any of the UI buttons to be able to toggle on/off these filters. I would suggest to remove those for now to keep the code simple. We can add them when/if we add those UI elements to the attack table.